easybuild/easyblocks/q/quantumespresso.py

```diff
@@ -415,11 +418,12 @@ def sanity_check_step(self):
 
         if 'pw' in targets or 'all' in targets:
             bins.extend(["dist.x", "ev.x", "kpoints.x", "pw.x", "pwi2xsf.x"])
-            if LooseVersion(self.version) >= LooseVersion("5.1"):
-                bins.extend(["generate_rVV10_kernel_table.x"])
-            if LooseVersion(self.version) > LooseVersion("5"):
-                bins.extend(["generate_vdW_kernel_table.x"])
-            else:
+            if LooseVersion(self.version) < LooseVersion("6.5"):
+                if LooseVersion(self.version) >= LooseVersion("5.1"):
+                    bins.extend(["generate_rVV10_kernel_table.x"])
```

Comment:
Hmm, doesn't 6.5 build the generators any longer?
So where does it take the tables from then??

Reply:
it generates them on-the-fly, they say it's fast (https://github.com/QEF/q-e/releases/tag/qe-6.5)

Reply:
Ok, Then I bet that they are using the code that can't stand opt levels above -O0 and can barely produce correct results even then....

I sort of promised them to fix the vdw generator several years ago but never got around to it...